### DISCUSSION TITLE
Simbrief Upload and MCDU

### DISCUSSION THREAD

**Discussion by Zimmerbz** [flybywiresim/aircraft#3234] [General]
Just a quick question, does the Simbrief upload actually load the plane with pax, fuel, cargo etc., or does it only enter the data to the MCDU and I have to manually enter those figures into the sim?  If it is the latter, will this change once the EFB is released and we can enter all of the figures through that?  Thanks

**Comment by Zimmerbz** [comment]
I hope that made sense, if you need me to clarify, please let me know

**Reply by 4791pg** [reply]
I'd like to know too
thx

**Reply by tracernz** [reply]
You can go to the PERF & WB pages in ATSU and load the SimBrief fuel and payload there (after an init request had already been done).

**Reply by lancealotg** [reply]
I am able to load the fuel only, the payload does not change for me. 
Is the payload working?
